src/controller/org.controller/index.js

```diff
@@ -167,21 +167,21 @@ router.get('/org/:identifier',
   /*
   #swagger.tags = ['Organization']
   #swagger.operationId = 'orgSingle'
-  #swagger.summary = "Retrieves the organization entry for the specified short name or UUID (accessible to all registered users)"
+  #swagger.summary = "Retrieves information about the organization specified by short name or UUID (accessible to all registered users)"
   #swagger.description = "
         <h2>Access Control</h2>
         <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
         <p><b>Regular, CNA & Admin Users:</b> Retrieves organization record for the specified shortname or UUID if it is the user's organization</p>
```

Comment:
I'd change this to 'Retrieves information about an organization' for consistency